### PR TITLE
📝(content) Add remaining content

### PR DESCRIPTION
**Pull request by Snugug**
- [x] Adds quotes to EN homepage
- [x] Add M100 blog post
- [x] Add footnotes to homepage stats

### CODE REVIEW

**Comment by Vanss472** [comment]
Hey @Snugug the only thing I see is the focus on the cta link

image

I would set the `background-color: transparent;`

**Comment by Snugug** [comment]
Thanks! I've resolved the banner images and the focus link, the Spanish content is coming in from elsewhere so we'll defer on that for now

**Comment by Vanss472** [comment]
cool!! Thanks.

Now I see a tiny laptop - news article
image

**Comment by Vanss472** [comment]
> > cool!! Thanks.
> > Now I see a tiny laptop - news article image
> 
> Is that on the stand alone page or the news page and its banner?

news article --> stand alone page https://cros-staging--ea7a9781-42b9-4102-8422-6f2951c71e50-ps3a7upm.web.app/en/posts/chromeos-100

File: site/en/posts/chromeos-100.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+## A refreshed ChromeOS.dev
+
+We’ve come a long way in the past decade, and it’s been incredibly rewarding to see how developers have used Chrome OS to grow their businesses and create innovative app experiences for users across devices.
+
+Every new feature we launch is designed to empower developers to keep growing and engaging wider audiences — no matter what they’re building or who they’re trying to reach. To make sure they have the resources they need to succeed, we created [ChromeOS.dev](https://chromeos.dev) in 2020 to give a home to all these new features, including code samples, best practices, technical guidance, and developer success stories.
+
+We're excited to announce ChromeOS.dev has a new look and a lot more great content on the way — including dedicated resources for developers building apps for education, and content for enterprise coming later this year. There will also be a series of posts focused on the foundational tech behind Chrome OS … and there might even be a fun Easter egg for developers to discover. So be sure to check out the site, explore the latest content, and look for clues.
```
Comment (Vanss472) [inline site/en/posts/chromeos-100.md:44]: EE *hint hint*

**Review by Vanss472 — DISMISSED** [review]

**Pull request merged**